<ui version="4.0" >
 <class>MyFirstForm</class>
 <widget class="QWidget" name="MyFirstForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string comment="Точно точно" >Мое первое приложение</string>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="4" row="0" column="2" colspan="3" >
    <widget class="QEidosTreeWidget" name="treeWidget" >
     <column>
      <property name="text" >
       <string>Наименование</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>ID</string>
      </property>
     </column>
    </widget>
   </item>
   <item rowspan="4" row="0" column="5" >
    <widget class="QTextEdit" name="textEdit" >
     <property name="autoFormatting" >
      <set>QTextEdit::AutoAll</set>
     </property>
    </widget>
   </item>
   <item row="0" column="6" >
    <widget class="QPushButton" name="ExitButton" >
     <property name="text" >
      <string>Выход</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6" >
    <widget class="QPushButton" name="GoButton" >
     <property name="text" >
      <string>Поехали</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6" >
    <widget class="QPushButton" name="GridButton" >
     <property name="text" >
      <string>GRID</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="4" >
    <widget class="QTableView" name="tableView" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>2</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="autoScrollMargin" >
      <number>5</number>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="sortingEnabled" >
      <bool>false</bool>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEidosTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>qeidostreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>MyFirstForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>310</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel" >
     <x>310</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
